src/marker.rs

```diff
@@ -0,0 +1,85 @@
+//! This module contains marker traits used by `wasm-bindgen` to verify its
+//! generated code.
+//!
+//! NOTE: This module is not exported as is. It is re-exported under
+//! `wasm_bindgen::__rt::marker`.
+//!
+//! # ⚠️ Unstable
+//!
+//! This is an internal module, no stability guarantees are provided. Use at
+//! your own risk.
+
+#[cfg(feature = "diagnostic")]
+extern crate rustversion;
```

Comment:
```suggestion
#[cfg(feature = "diagnostic")]
use rustversion::rustversion;
```
This doesn't work?

Reply:
```
error[E0432]: unresolved import `rustversion::rustversion`
 --> src\marker.rs:2:5
  |
2 | use rustversion::rustversion;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^ no `rustversion` in the root
```

Reply:
Ah! The macro isn't actually called `rustversion`, `attr` is already the name of the proc-macro. So accessing it via `rustversion::attr` works just fine.
This line should be fine to remove entirely!